docs: initial start of the maintainership docs under the bounty program (#1894)
asyncapi/community: docs/maintainership-guide/general-information/contributor/expectations.md

---
title: Contributor Expectations
weight: 80
---

> A successful mentorship program doesn't just happen —like any learning and development program, it depends on intentional planning, mutual trust, and active participation. While the program is designed to support and empower contributors, its success hinges on your commitment and accountability.

Reviewer: ```suggestion
A successful mentorship program doesn't just happen; it depends on intentional planning, mutual trust, and active participation. While the program is designed to support and empower contributors, its success hinges on your commitment and accountability.
```

- **Follow standards:** Adhere to AsyncAPI organization's coding, contribution, and documentation guidelines to maintain quality and consistency.
-  **Be open to feedback:** Embrace constructive input from mentors, peers, and community members to refine your work.
- **Support fellow contributors:** Celebrate each other’s wins, and offer help when possible. Collaboration over competition is the AMP way.

### Commitment

Your project is your top priority during the mentorship phase. You’re expected to dedicate **at least 30 hours per week for 12 weeks** to ensure meaningful progress and learning.

That said, we understand that life happens. Unexpected events like illness, power outages, internet issues, or family emergencies can impact your availability. The key is **transparency and proactive communication.**

If you experience anything that might affect your ability to stay on track, here’s what you should do:

- **Keep your mentor informed:** Notify your mentor as soon as possible if you’ll be unavailable, even for a short period.

> - **Set realistic expectations:** If 30 hours/week isn’t feasible, work with your mentor early on to define a manageable and consistent commitment.

Reviewer: ```suggestion
- **Set realistic expectations:** If 30 hours per week isn’t feasible, work with your mentor early on to define a manageable and consistent commitment.
```

> - **Manage your time wisely:** Plan around project milestones and meet deadlines to keep your progress steady.

Reviewer: ```suggestion
- **Manage your time wisely:** Plan your tasks around project milestones and meet deadlines to maintain steady progress.
```
